templates/citizen_action_manager/contact_participants.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends 'citizen_action/_layout.html.twig' %}
+
+{% block page_title 'Contacter des inscrits à l\'a action citoyenne' %}
+
+{% block javascripts %}
+    {% import 'javascript.js.twig' as js %}
+
+    <script src="{{ asset('ckeditor/ckeditor.js') }}"></script>
+    <script src="{{ asset('ckeditor/lang/fr.js') }}"></script>
+    <script type="text/javascript">
+        Kernel.onLoad(function() {
+            {{ js.ckeditor('message') }}
+        });
+    </script>
+{% endblock %}
+
+{% block citizen_action_content %}
+    <main class="committee__event__create">
+
+        <section class="l__wrapper--slim space--30-0 text--body">
+            <div class="b__nudge--bottom-huge">
+                <h2 class="b__nudge--bottom-small text--medium">
+                    Contacter des inscrits à l'action citoyenne
+                </h2>
+                <div class="b__nudge--bottom-medium">
+                    {% if contacts is empty %}
+                        <div class="error">
+                            Vous n'avez pas selectionné de participant destinataire de ce message,
+                            il ne pourra donc pas être envoyé.
+                        </div>
+                    {% else %}
+                        <div class="text--body">
+                            {% set msg = 'Votre message sera envoyé par e-mail à %s de l\'action citoyenne.' %}
+                            {{ msg|format('event.participants_count'|transchoice(contacts|length)) }}
+                        </div>
+                    {% endif %}
+                </div>
+
+                {{ form_start(form) }}
+
+                    {{ form_errors(form.message) }}
+
+                    <input type="hidden" name="contacts" value="{{ contacts|json_encode }}" />
```

Comment:
This could be done in the FormType no ? With the `buildView...` methods no ? I guess it's not a big deal to keep it here but it would be at least better to dynamically set the name of the input no ?